packages/server/src/api/controllers/role.ts

```diff
@@ -3,6 +3,7 @@ import { getUserMetadataParams, InternalTables } from "../../db/utils"
 import { Database, Role, UserCtx, UserRoles } from "@budibase/types"
 import { sdk as sharedSdk } from "@budibase/shared-core"
 import sdk from "../../sdk"
+import { BUILTIN_ROLE_IDS } from "@budibase/backend-core/src/security/roles"
```

Comment:
can't pull in references this way - can break builds - can you update this to `import { roles } from @budibase/backend-core`.